crud/src/main/java/com/redhat/lightblue/rest/crud/AbstractCrudResource.java

```diff
@@ -111,7 +122,7 @@ public Response getSearchesForEntity(@PathParam("entity") String entity,
         }
         CallStatus st=new FindCommand(freq.getEntityVersion().getEntity(),
                                       freq.getEntityVersion().getVersion(),
-                                      freq.toJson().toString()).run();
+                                      freq.toJson().toString(), metrics).run();
```

Comment:
I wrote that comment before I made the changes to track errors via response object. With response object based tracking, the Bulk requests can be handled easily as well as mediator errors.